tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/LazilyInitializedMovingAverages.java

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Not a great idea to couple this change with a library upgrade, however it's the piece I'm most interested in leveraging